src/Command/CheckCommand.php

```diff
@@ -76,19 +76,24 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $drupalFinder = new DrupalFinder();
-        $path = realpath($input->getArgument('path'));
 
-        if (!$path || !file_exists($path)) {
-            $output->writeln(sprintf('<error>%s does not exist</error>', $input->getArgument('path')));
-            return 1;
+        $paths = [];
+        foreach ($input->getArgument('path') as $path) {
+            $realPath = realpath($path);
+            if (!$realPath) {
+                $output->writeln(sprintf('<error>%s does not exist</error>', $path));
+                return 1;
+            }
+
+            $paths[] = $realPath;
         }
 
-        $drupalFinder->locateRoot($path);
+        $drupalFinder->locateRoot($paths[0]);
         $this->drupalRoot = $drupalFinder->getDrupalRoot();
         $this->vendorRoot = $drupalFinder->getVendorDir();
 
         if (!$this->drupalRoot) {
-            $output->writeln('Unable to determine the Drupal root');
+            $output->writeln(sprintf('<error>Unable to locate the Drupal root in %s</error>', $paths[0]));
```

Comment:
In case somebody does provide multiple paths and the first one is not within Drupal, we'll want to let them know which of the paths we were looking at.